images/scripts/debian.setup

```diff
@@ -130,6 +131,18 @@ Pin-Priority: -1
 Package: thin-provisioning-tools
 Pin: release a=unstable
 Pin-Priority: 10
+
+Package: virtinst
```

Comment:
Let's make sure we don't forget to take these out.

Reply:
There's not much harm to it -- once the packages enter testing again, the choice of priorities will install it from there again, and this will just be dead code.